python/lsst/jointcal/jointcal.py

```diff
@@ -184,7 +191,7 @@ def run(self, ref, tract):
         andConfig = AstrometryNetDataConfig()
         andConfigPath = os.path.join(anDir, "andConfig.py")
         if not os.path.exists(andConfigPath):
-            raise RuntimeError("astrometry_net_data config file \"%s\" required but not found" %andConfigPath)
+            raise RuntimeError("astrometry_net_data config file \"%s\" required but not found"%andConfigPath)
```

Comment:
Yup: it's skirting right up against the limit. However, this should go away as part of DM-5205.